jskit/encoders/utils/config.cfg

```diff
@@ -1,14 +1,14 @@
 [TRAIN_PARAMETERS]
 max_contexts_length = 128
-MAX_CANDIDATE_LENGTH = 64
-train_batch_size = 32
-eval_batch_size = 2
-MAX_HISTORY = 4
+max_candidate_length = 64
+train_batch_size = 8
+eval_batch_size = 1
+max_history = 4
```

Comment:
are you sure this should be lower case? if you look at `train.py` its all reading using upper case? can you confirm and make sure train/infer work?

Reply:
Yes, it does work